OpenSim/Sandbox/MatlabScripts/simple_arm.m

```diff
@@ -0,0 +1,98 @@
+% ----------------------------------------------------------------------- %
```

Comment:
Use a consistent license header; the header for the other script includes the file name; whereas this one does not.